src/features/todos.ts

```diff
@@ -1,8 +1,21 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSelector, createSlice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
+import { RootState } from '../app/store';
 
 export const todosSlice = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    set: (_, action) => {
+      return action.payload;
```

Comment:
The `set` reducer directly assigns the `action.payload` to the state. Ensure that the `payload` is always an array of `Todo` objects to maintain type safety. Consider adding type checks or validation if necessary.